features/cron.feature

```diff
@@ -189,13 +189,6 @@ Feature: Manage WP-Cron events and schedules
       | name   | interval |
       | hourly | 3600     |
 
-  Scenario: Testing WP-Cron
-    When I try `wp cron test`
-    Then STDERR should not contain:
-      """
-      Error:
-      """
-
```

Comment:
Instead of deleting this test, I think it should stay for a valid setup check, and we need to add a second test after deleting the `wp-cron.php` to verify that it does indeed throw an error.

Reply:
As I mentioned in the issue description, this scenario performs a request to `example.com` over the internet (which returns a 404 because example.com doesn't use WordPress).

@schlessera Does WP-CLI contain any existing tests which perform an HTTP request to an actual local WordPress site?

Reply:
You could probably point to the proper WordPress instance using something like this as setup and populating the options accordingly:
```
Given a WP install
And I launch in the background `wp server --host=localhost --port=8080`
```